staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -140,6 +140,12 @@ const (
 	//
 	// Deprecates and removes SelfLink from ObjectMeta and ListMeta.
 	RemoveSelfLink featuregate.Feature = "RemoveSelfLink"
+
+	// owner: @shaloulcy
+	// alpha: v1.18
+	//
+	// Allow index labels and fields in apiserver to accelerate list operation.
```

Comment:
nit:
Allows label and field based indexes in apiserver watch cache to accelerate list operations.